csrc/scheduler/matmul.cpp

```diff
@@ -30,6 +30,17 @@ MatmulScheduler::MatmulScheduler(
 
 void MatmulScheduler::schedule(Fusion* fusion) {
   FUSER_PERF_SCOPE("Schedule Matmul Fusion");
+  // Skip scheduling if Matmul will be expression evaluated.
+  if (isOptionEnabled(EnableOption::MatmulExprEval)) {
+    NVF_CHECK(fusion->outputs().size() == 1)
```

Comment:
So right now we are asserting on a single output from the fusion? cc'ing @protonu This might be a sharp edge that you'd want to keep in mind.

Reply:
Thanks!